src/library_wasmfs.js

```diff
@@ -86,7 +86,34 @@ var WasmfsLibrary = {
       // User code should not be using FS.cwd().
       // For file preloading, cwd should be '/' to begin with.
       return '/';
-    }
+    },
+
+#if FORCE_FILESYSTEM
+    // Full JS API support
+    mkdir: function(path, mode) {
+      // Adjust mode in the same way as the JS FS
+      mode = mode !== undefined ? mode : 511 /* 0777 */;
+      mode &= {{{ cDefine('S_IRWXUGO') }}} | {{{ cDefine('S_ISVTX') }}};
+      mode |= {{{ cDefine('S_IFDIR') }}};
```

Comment:
This logic is already handled inside WasmFS.
```suggestion
```

Reply:
Good catch! Removing.